yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html

```diff
@@ -19,14 +19,14 @@
       <div class= "border">
       {% if edit %}
         <div class="edit_button">
+        <form method="get" action="{% url 'edit_page' workflow.id version.id %}">
```

Comment:
you need to close the `<div class="edit_button">`

If you were having issues, it's because in line 64 we have an invalid closing div tag. It on line 64 it looks like `<div/>' when it should be '</div>'. 

If you make that fix, I'll close my pull request that does the same things it looks like you've already done